index.html

```diff
@@ -0,0 +1,220 @@
+            document.querySelector("#tasks").appendChild(newTask);
+            
+            let input = document.createElement('input');
+            input.setAttribute("type", "checkbox");
+            input.setAttribute("id", "input_"+ tasks.indexOf(value));
+            input.classList.add("checkbox")
+            input.onchange = function(){
+                crossTask(value);
+            };
+            document.querySelector("#div_" + tasks.indexOf(value)).appendChild(input);
+
+            let task = document.createElement('label');
+            task.innerText = value.name;
+            task.setAttribute("id", tasks.indexOf(value));
+            task.style.float = "left";
+            document.querySelector("#div_" + tasks.indexOf(value)).appendChild(task);
+
+            let deletebtn = document.createElement('button');
+            deletebtn.innerHTML = '<i class = "fa fa-close"></i>';
+            deletebtn.setAttribute("id", "deletebtn");
+            deletebtn.onclick = function(){
+                deleteTask(value);
+            };
+            document.querySelector("#div_" + tasks.indexOf(value)).appendChild(deletebtn);
+
+            if(value.completed){
+                task.style.textDecoration = "line-through"
+                input.checked = true;
+            }else{
+                numOfActiveTasks++;
+            }
+        }
+
+        function deleteTask(value){
+            let deleted = [];
+            let index = tasks.indexOf(value);
+            for(let i = 0; i < tasks.length; i++){
+                if(index != i){
+                    deleted.push(tasks[i])
+                }
+            }
+
+            tasks = deleted;
+            localStorage.removeItem("tasks");
+            localStorage.setItem("tasks", JSON.stringify(tasks));
+            location.reload();
+        }
+
+        function crossTask(value){
+            if(document.getElementById("input_" + tasks.indexOf(value)).checked){
+                tasks[tasks.indexOf(value)].completed = true;
+                localStorage.setItem("tasks", JSON.stringify(tasks));
+            }
+            else{
+                tasks[tasks.indexOf(value)].completed = false;
+                localStorage.setItem("tasks", JSON.stringify(tasks));
+            }
+            reloadNumbOfTasks();
+            checkFilter();
+        }
+
+        function checkFilter(){
+            let btnAll = document.getElementById("all");
+            let btnComplited = document.getElementById("completed");
+            let btnActive = document.getElementById("active");
+
+            if(btnAll.classList.contains("active")){
+                allTasks();
+            }else if(btnComplited.classList.contains("active")){
+                completedTasks();
+            }else if(btnActive.classList.contains("active")){
+                activeTasks();
+            }
+        }
+
+        function reloadNumbOfTasks(){
+            let numb = document.getElementById('numbOfActive');
```

Comment:
you already have this reference at the beginning of the file and it is not changing during the app lifetime